scripts/fonts/add-polish-glyphs.py

```
@@ -0,0 +1,98 @@
+for key in ("Family", "Fullname", "PostScriptName", "Preferred Family"):
+    f.appendSFNTName("English (US)", key, "Czikago")
+# the original file carried a stale "All Rights Reserved" string; replace it with
+# the actually-accepted provenance (Casady's ChicagoFLF is released as public domain)
+f.copyright = ("Czikago: ChicagoFLF (Robin Casady, public domain), a revival of "
+               "Chicago by Susan Kare, extended with Polish diacritics.")
+f.appendSFNTName("English (US)", "License",
+                 "Public domain, per Robin Casady's ChicagoFLF release.")
+
+for ext in ("woff", "woff2"):
+    f.generate("public/fonts/Czikago.%s" % ext)
+os.makedirs("/tmp/fontpatch/out", exist_ok=True)
```

Ruff (0.15.17)

[error] Probable insecure usage of temporary file or directory: "/tmp/fontpatch/out"

(S108)